pkg/scaling/executor/scale_jobs.go

```diff
@@ -227,3 +224,67 @@ func (e *scaleExecutor) getFinishedJobConditionType(j *batchv1.Job) batchv1.JobC
 	}
 	return ""
 }
+
+// NewScalingStrategy returns ScalingStrategy instance
+func NewScalingStrategy(logger logr.Logger, scaledJob *v1alpha1.ScaledJob) ScalingStrategy {
+	switch scaledJob.Spec.ScalingStrategy {
+	case "custom":
+		logger.Info("Selecting Scale Strategy", "specified", scaledJob.Spec.ScalingStrategy, "selected:", "custom", "customScalingQueueLength", scaledJob.Spec.CustomScalingQueueLengthDeduction, "customScallingRunningJobPercentage", scaledJob.Spec.CustomScalingRunningJobPercentage)
```

Comment:
I'd convert this message to be a DEBUG one, so we don't flood the output with log messages.:
`logger.V(1).Info(..)`

Reply:
Done